src/solc_0.8/assetERC1155/AssetERC1155.sol

```diff
@@ -5,9 +5,6 @@ import "./AssetBaseERC1155.sol";
 
 // solhint-disable-next-line no-empty-blocks
 contract AssetERC1155 is AssetBaseERC1155 {
-    uint256 private constant CHAIN_INDEX_OFFSET_MULTIPLIER = uint256(2)**(256 - 160 - 1 - 8);
-    uint256 public constant CHAIN_INDEX = 0x00000000000000000000000000000000000000007F8000000000000000000000;
```

Comment:
won't that break the storage after upgading ?

Reply:
No - moved to Lib